dbgpt/model/adapter/embeddings_loader.py

```diff
@@ -50,6 +50,17 @@ def load(self, model_name: str, param: BaseEmbeddingModelParameters) -> Embeddin
                 if proxy_param.proxy_backend:
                     openapi_param["model_name"] = proxy_param.proxy_backend
                 return OpenAPIEmbeddings(**openapi_param)
+            elif model_name in ["proxy_tongyi"]:
+                from dbgpt._private.config import Config
+                from dbgpt.rag.embedding import TongYiEmbeddings
+
+                CFG = Config()
+                proxy_param = cast(ProxyEmbeddingParameters, param)
+                tongyi_param = {}
+                tongyi_param["api_key"] = CFG.tongyi_proxy_api_key
+                if proxy_param.proxy_backend:
+                    tongyi_param["model_name"] = proxy_param.proxy_backend
+                return TongYiEmbeddings(**tongyi_param)
```

Comment:
The model_name default value will be `text-embedding-ada-002`

Reply:
ok. I have add tongyi config in .env.template